charts/gardener/controlplane/charts/runtime/templates/admission-controller/secret-cert.yaml

```diff
@@ -0,0 +1,17 @@
+{{- if .Values.global.admission.enabled }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: gardener-admission-controller-cert
+  namespace: garden
+  labels:
+    app: gardener
+    role: admission-controller
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+type: Opaque
```

Comment:
Tbh, neither am I sure about the benefits or consequences nor have I found something during a quick research except that is has some benefits for `kubectl`. To be sure, I'd like to stick to `generic` for now since we use it in other charts as well if you agree?

Reply:
Yep, agreed.